gemini/master.py

```diff
@@ -32,25 +33,26 @@ def info() -> str:
 @app.route('/v1/nodes/join', methods=['POST'])
 def join() -> str:
     node_id = generate_node_id()
-    token = generate_node_token()
+    password = generate_node_password()
 
     value = {
         'node_id': node_id,
         'expires': 5000,
```

Comment:
we can get rid of this expires thing for now, from the consul entry and the response